profiles/common/modules/features/ec_world_countries/nuts_regions/tests/nuts_regions.test

```diff
@@ -10,7 +10,9 @@
  */
 class NutsRegionsTestCase extends DrupalWebTestCase {
 
-  // $adminUser is a user with permission to administer blocks.
+  /**
+   * $adminUser is a user with permission to administer blocks.
```

Comment:
Normally that should be comment like below:
```php
/**
 * User with permission to administer blocks.
 *
 * @var stdClass
 */
```
or

```php
/**
 * Drupal user with permission to administer blocks.
 *
 * @var stdClass
 */
```
It is formatted as it elsewhere.